aries-site/src/examples/templates/feedback/PowerOnExample.js

```diff
@@ -0,0 +1,205 @@
+        </Footer>
+      </Box>
+    </Layer>
+  );
+};
+
+export const PowerOnExample = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isSuccessful, setIsSuccessful] = useState(false);
+  const [rating, setRating] = useState();
+  const [showFeedback, setShowFeedback] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (isSuccessful) {
+        setShowFeedback(true);
+      }
+    }, 1000);
+  }, [isSuccessful]);
+
+  // simulating a call to a power on API call. Demo purposes
+  const PowerOnLoading = () => {
+    setIsLoading(true);
+    setTimeout(() => {
+      setIsLoading(false);
+      setIsSuccessful(true);
+    }, 3000);
+  };
+
+  // fetch call to POST api should be in onSubmit
+  // for demo purposes disabling actual API call
+  // submitFeedback();
+  const onSubmit = useCallback(value => {
+    const data = {
+      values: {
+        // url for the route which the feedback survey is presented
+        fullURL: url,
+        // type of device. Possible values 'desktop', 'tablet', 'mobile'.
+        deviceType: deviceType,
+        // QIDs are unique to each feedback instance and will be provided when
+        // coordinating with program manager, Doris Singer (doris.singer@hpe.com).
+        QID1: value.value['power-rating-example'],
+        QID2_TEXT: value.value['power-textArea-example'],
+      },
+    };
+    // dev uses POST https://api.qualtrics.com/{survey-id}-create-a-new-response
+    fetch(API_URL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+        'X-API-TOKEN': API_TOKEN,
+      },
+      body: JSON.stringify(data),
+    })
+      .then(response => response.json())
+      .catch(error => {
+        // Error handling here
+      });
+  }, []);
+
+  return (
+    <Card width="large" gap="medium" justify="center" pad="medium">
+      <Box>
+        <Heading level={1} size="small" margin="none">
+          Power On
+        </Heading>
+        <Anchor href="#" label="Learn more about server actions" />
+      </Box>
+      {!isSuccessful ? (
+        <Text size="large">You are powering on 1 device</Text>
+      ) : (
+        <Box align="center" gap="xsmall" direction="row">
+          <StatusGoodSmall color="green" />
+          <Text size="large">Power On initiated for 1 device.</Text>
+        </Box>
+      )}
+      {!isLoading && !isSuccessful && (
+        <Text size="large">FTCUSAMountain_ML350Gen10_testcluster</Text>
+      )}
+      {isLoading && (
+        <Spinner
+          alignSelf="center"
+          size="medium"
+          message={{
+            start: 'Loading more data.',
+            end: 'Data has been loaded.',
+          }}
+        />
+      )}
+      {!isSuccessful && (
+        <Box
+          justify="end"
+          gap="medium"
+          margin={{ top: 'medium', bottom: 'small' }}
+          direction="row"
+        >
+          <Button label="Cancel" />
```

Comment:
I have been seeing the word "dismiss" used in these feedback contexts.

```suggestion
          <Button label="Dismiss" />
```